DigitalLearningSolutions.Web/Styles/shared/cardWithButtons.scss

```diff
@@ -4,10 +4,9 @@
   background-color: #fff;
   @extend .nhsuk-u-margin-top-3;
   @extend .nhsuk-u-margin-bottom-4;
-  padding-top: 8px;
   padding-bottom: 16px;
-  padding-left: 32px;
-  padding-right: 32px;
+  border: 1px solid $nhsuk-border-color;
+  border-bottom-width: 4px;

```

Comment:
Might be worth listing on the ticket which cards are affected so the tester can sanity check they're all ok :)